src/pybkgmodel/data.py

```diff
@@ -501,23 +543,235 @@ def load_events(cls, file_name, cuts):
+        """
+
+        data_names_mapping = {
+            'EVENT_ID': 'daq_event_number',
+            'RA': 'event_ra',
+            'DEC': 'event_dec',
+            'GAMMANESS': 'gammaness',
+            'ENERGY': 'event_energy',
+        }
+
+        with fits.open(file_name, memmap=False) as input_file, \
+            erfa_astrom.set(ErfaAstromInterpolator(1 * u.s)):
+            try:
+                evt_hdu = input_file["EVENTS"]
+                evt_head = evt_hdu.header
+                evt_data = Table.read(evt_hdu)
+
+                event_data = {}
+
+                for key, name in data_names_mapping.items():
+                    if key in evt_data.keys():
+                        event_data[name] = evt_data[key].quantity
+
+                        if not evt_data[key].unit:
+                            event_data[name] *= u.one
+
+                # Event times need to be converted from Instrument reference epoch
+                ref_epoch = astropy.time.Time(evt_head['MJDREFI'],
+                                              evt_head['MJDREFF'],
+                                              scale=evt_head['TIMESYS'].lower(),
+                                              format='mjd'
+                                              )
+
+                evt_time = evt_data['TIME'].quantity + ref_epoch
+                event_data['mjd'] = evt_time.mjd * u.d
+
+                # TODO: current observatory location only La Palma, no mandatory header keyword
+                obs_loc  = EarthLocation(lat=28.761758*u.deg,
+                                        lon=-17.890659*u.deg,
+                                        height=2200*u.m)
+
+                if evt_head['OBS_MODE'] in ('POINTING', 'WOBBLE'):
+
+                    alt_az_frame = AltAz(obstime=evt_time,
+                                        location=obs_loc)
+
+                    coords = SkyCoord(evt_head['RA_PNT'] *u.deg,
+                                    evt_head['DEC_PNT'] *u.deg,
+                                    frame='icrs')
+
+                    altaz_pointing =  coords.transform_to(alt_az_frame)
+
+                    event_data['pointing_zd'] = 90 * u.deg - altaz_pointing.alt
+                    event_data['pointing_az'] = altaz_pointing.az.to(u.deg)
+
+
+                    event_data['pointing_ra'] = np.array(
+                                                            [evt_head['RA_PNT']] \
+                                                            * len(event_data['pointing_zd'])
+                                                            ) * u.deg
+                    event_data['pointing_dec'] = np.array(
+                                                            [evt_head['DEC_PNT']] \
+                                                            * len(event_data['pointing_zd'])
+                                                            ) * u.deg
+
+                elif evt_head['OBS_MODE'] == 'DRIFT':
```

Comment:
Just raise an error to let users know this is not supported instead of providing preliminary or invalid data (same for the next else block, this should be an actual error, not a print statement)